cockroachDB/query.js

```diff
@@ -0,0 +1,25 @@
+var pool = require("./connection.js").pool;
+
+function query(query, values, cb) {
+  pool.connect(function(err, client, done) {
+    // Close communication with the database and exit.
+    var finish = function() {
+      done();
+      // process.exit();
+    };
+
+    if (err) {
+      console.error("could not connect to cockroachdb", err);
+      finish();
+    }
+
+    client.query(query, values, (err, res) => {
+      if (err) cb(err, null);
+      console.log(res);
+      cb(null, res);
+      finish();
+    });
+  });
+}
+
+module.exports.query = query;
```

Comment:
this is used for each query, it deals with pool connections as per cockroach docs